src/main/scala/com/minosiants/pencil/LiteralSyntaxMacros.scala

```diff
@@ -79,7 +79,7 @@ object LiteralSyntaxMacros {
     singlePartInterpolator(c)(
       args,
       "Attachment",
-      Attachment.fromString(_).isRight,
+      Attachment.fromString[cats.effect.IO](_).unsafeRunSync.isRight, // We're touching files in macros?
```

Comment:
We check if a file exist otherwise it can not be sent

Reply:
Right, but macros happen at compile time. So the check is based on compile time, not runtime status of the file. If I moved the binary to another machine and the file does not exist there, wouldn't this entire macro behavior fail?

Reply:
Fair point. I will remove checking of the file existence.